CMakeLists.txt

```diff
@@ -369,3 +369,19 @@ install(
   RUNTIME DESTINATION bin
   BUNDLE DESTINATION bin
   LIBRARY DESTINATION lib)
+
+# for creating DEB and RPM packages
+set(CPACK_PACKAGE_CONTACT "ruby.mamistvalove@gmail.com")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY
+    "Notes is a simple and beautiful note taking app.")
+set(CPACK_PACKAGE_DESCRIPTION
+    "Notes is a simple and beautiful note taking app. It is cross-platform and written in C++ using the Qt framework."
+)
+# set dependencies based on the Qt version
+if(${QT_VERSION_MAJOR} EQUAL 6)
+  set(CPACK_DEBIAN_PACKAGE_DEPENDS "")
```

Comment:
So i guess we can do it like this? Still trying to figure out which packages I'm supposed to put here. Asking ChatGPT... :D

Reply:
That's how I'd do too.

Reply:
For Qt 5 we have those already listed at https://github.com/nuttyartist/notes/blob/master/packaging/linux/debian/control, and this variable appears to take the same format.